README.md

```diff
@@ -1,15 +1,12 @@
+# Gatsbyn Suomenkielinen Käännös
```

Comment:
Title Case is not a thing in Finnish language.
```suggestion
# Gatsbyn suomenkielinen käännös
```

Reply:
Yeah, I did it intentionally before to follow other translations, but I'll fix it 😃.